google-cloud-gemserver.gemspec

```diff
@@ -3,9 +3,17 @@
 lib = File.expand_path("../lib", __FILE__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 
+dirs = ["bin", "lib", "acceptance", "integration", "test", "secrets", "docs"]
+files = [
+  "google-cloud-gemserver.gemspec", "Gemfile", "Gemfile.lock", "Rakefile",
+  "CONTRIBUTING.md", "README.md", "LICENSE", ".travis.yml", ".yardopts",
+  ".rubocop.yml"
```

Comment:
Remove the development-related files. You also don't need the gemspec in the gem itself. I think all you need are the `*.md` files, `LICENSE`, and `.yardopts`.